autoload/neomake/makers/ft/haskell.vim

```diff
@@ -1,5 +1,6 @@
 
 function! neomake#makers#ft#haskell#MakerAvailable(command)
```

Comment:
This function is script-local and should be renamed to `s:MakerAvailable` for clarity.